Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -534,9 +539,12 @@ void GraphicsContextSkia::setLineCap(LineCap lineCap)
     m_skiaState.m_stroke.cap = toSkiaCap(lineCap);
 }
 
-void GraphicsContextSkia::setLineDash(const DashArray&, float /*dashOffset*/)
+void GraphicsContextSkia::setLineDash(const DashArray& dashArray, float dashOffset)
 {
-    notImplemented();
+    ASSERT(!(dashArray.size() % 2));
+
+    m_skiaState.m_dash.array = dashArray;
```

Comment:
If so then you'd change this line to
```suggestion
    m_skiaState.m_dash.array = WTFMove(dashArray);
```